packaging/installer/kickstart.sh

```diff
@@ -20,12 +20,26 @@ KICKSTART_SOURCE="$(
     echo "$(pwd -P)/${self##*/}"
 )"
 DEFAULT_PLUGIN_PACKAGES=""
-PATH="${PATH}:/usr/local/bin:/usr/local/sbin"
 REPOCONFIG_DEB_VERSION="4-1"
 REPOCONFIG_RPM_VERSION="4-1"
 START_TIME="$(date +%s)"
 STATIC_INSTALL_ARCHES="x86_64 armv7l armv6l aarch64 ppc64le"
 
+# ======================================================================
+# Properly sort out inconsistencies in `$PATH` across distros
+#
+# Debian and Ubuntu still don’t include sbin directories for regular
+# users (even though the distinction is all but pointless at this point in
+# time), and a number of distros don’t include the standard paths under
+# `/usr/local` either.
+
+for dir in /usr/sbin /sbin /usr/local/bin /usr/local/sbin ; do
+  case ":${PATH}:" in
+    *:${dir}:*) ;;
+    *) PATH="${PATH}:${dir}" ;;
+  esac
+done
```

Comment:
If PATH is empty, it will start with `:`. We can ignore this since in practice an empty PATH may not be possible, right?

Reply:
It should not be possible unless the user does something very stupid.